src/status.cc

```diff
@@ -118,6 +118,13 @@ std::string getStatusString(Status status) {
       return "[k] field is missing for an HMAC key";
     case Status::JwksHMACKeyBadK:
       return "[k] field is not string for an HMAC key";
+
+    case Status::JwksX509BioWriteError:
+      return "X509 BIO_Write function fails";
```

Comment:
Not a fan of this error name or message. An API producer / end user doesn't know what the `BIO_Write` function is.

Reply:
these errors should be rare, But in case, it indeed happen,  we want to know it,  so let us call it "internal error"